src/drawers/rectDrawer.ts

```diff
@@ -33,7 +33,8 @@ export module _Drawer {
       return this._labelsTooWide;
     }
 
-    public drawText(data: any[], attrToProjector: AttributeToProjector, userMetadata: any, plotMetadata: Plot.PlotMetadata) {
+    public drawText(data: any[], attrToProjector: AttributeToProjector, boundingBox: D3.Selection, userMetadata: any,
+        plotMetadata: Plot.PlotMetadata) {
```

Comment:
I tried it out and apparently we can't because the renderArea encapsulates the entire "bar render area" and does not actually match the visible area.

![image](https://cloud.githubusercontent.com/assets/1175041/6946597/5bebdbd6-d856-11e4-9aae-55d7a7e7925e.png)